src/pydrex/diagnostics.py

```diff
@@ -0,0 +1,231 @@
+import itertools as it
+
+import numpy as np
+from numpy import random as rn
+import scipy.linalg as la
+
+import pydrex.tensors as _tensors
+
+
+def Bingham_average(orientations, axis="a"):
```

Comment:
Probably best to keep the function name lower-cased (my previous comment was mainly referring to the docstring, even though I could have made it clearer).